rules/watch-build.mjs

```diff
@@ -0,0 +1,23 @@
+import fs from "fs";
```

Comment:
This is great! Do you want to update `npm run watch` and maybe `npm run create-rule`? ;)